### PR TITLE
Lily/create about me navbarlink

### LINKED ISSUE #53: Navbar - Implement CSS for About Us link
# Requirements 

- The About us Page should be positioned and styled matching the wireframe below
-  Page and Link should be responsive 

## Wireframe reference

Untitled

### PR DESCRIPTION
**Pull request by Kumaguro3**
Resolves #53 

# What changed 

Changed position of About Us link to absolute to have the element on the right as the nav is originally set with justify-between. Added also gap, padding, and margin. I also used "items center" to center the elements for a good & responsive spacing inside navbar. 

 Good to know: Responsive mainly on bigger devices(medium and above) as I had to "hide" the burger icon as this link was not originally included in it when the page open on smaller devices and the links break into the burger icon. 

So someone will need to include the nav links into the burger when we browse on smaller windows 

EDIT: 
- Fixed the double quote issue and dangling commas and run linter on other files. 
- Deleted old yarn release 

# Testing instructions 

run `yarn lint --fix `

### CODE REVIEW

**Review by theyokohamalife — COMMENTED** [review]
@Kumaguro3 Thank you for helping out! It looks like there are some linting changes included in the PR. Did you run `yarn prepare`? It's preferable to limit the scope of the PR to just the CSS changes. Also it looks like there is a file conflict. Please make sure you're working off the latest main. 🙇🏾‍♀️

**Comment by Kumaguro3** [comment]
Thanks for your kind feedback! :) I forgot to do Yarn prepare but it's now on my computer.

Here a few questions: 

"It's preferable to limit the scope of the PR to just the CSS changes" Does this mean I accidentally changed a few things as I didn't have Yarn Prepare ? As I worked on the css I am not sure what accidental changes I made :(

After I did Yarn prepare, does it automatically fix all the accidental changed that it made perhaps ? And it will be all good once I commit and push again?

Regarding the conflict files, unfortunately I don't really understand... Do you mean I was not supposed to create a new branch and just work from default " branch main "? 

Screenshot 2022-10-26 at 13 00 55

File: components/TopNav.vue [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
+                />
             </div>
         </div>
         <LocaleSelector />
     </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue'
+import { defineComponent } from "vue";
```
Comment (ann-kilzer) [inline components/TopNav.vue:26]: To jump in on LaShawn's comment, we don't want to introduce style changes between different code authors, so we use a tool called eslint for consistency. Notice you accidentally changed the quotes here, and that distracts from the meaningful code changes. Please run both `yarn prepare` and `yarn lint --fix`

Thanks 🙏

Comment (ann-kilzer) [reply components/TopNav.vue:26]: Yarn prepare is a command you only need to run once, to set up githooks that check the code on push. Yarn lint is a command you'll run often

**Comment by theyokohamalife** [comment]
> Regarding the conflict files, unfortunately I don't really understand... Do you mean I was not supposed to create a new branch and just work from default " branch main "?
> 
Hey @Kumaguro3 ! I think @ann-kilzer covered how to use `yarn prepare` and `yarn lint` really well to avoid unintentional style changes. I just wanted to explain a bit more about the file conflict. If you notice on this PR it says you have a conflict with the `TopNav.vue` file.

Screen Shot 2022-10-26 at 10 41 03 PM

This means that this file has probably been updated since you created a copy on your branch. If you click that `Resolve conflicts` button you'll be able to see what's different between the updated file and your file. You can also pull the latest main and resolve the conflicts in your code editor, but it's a little more complex. If you'd like to pair on it, just send me a message on Slack :)

**Review by ermish — APPROVED** [review]
LGTM! :shipit:  thanks for the contribution @Kumaguro3

**Pull request merged**